### DISCUSSION TITLE
Will reddit functionality be affected by the upcoming API changes?

### DISCUSSION THREAD

**Discussion by drag083** [AAndyProgram/SCrawler#130] [Q&A] [answered]
See https://reddit.com/r/reddit/comments/12qwagm/an_update_regarding_reddits_api/

**Comment by AAndyProgram** [comment, marked as answer]
I read about it a month ago. As far as I understand, this update just enhance the capabilities for devs who need more and will not affect the current API. Anyway, when it happens, we'll see.
